climate-precip-avg.py

```diff
@@ -0,0 +1,40 @@
+import sys
+import netCDF4
+from netCDF4 import Dataset
+import xarray as xr
+import numpy as np
+import pandas as pd
+from matplotlib import pyplot as plt
+np.set_printoptions(threshold=np.inf)
+
+file = '~/dev/climate-datasets/Berkeley/gpcc_10_combined.nc'
+nc = xr.open_dataset(file, decode_times=False)
+nc
+
+dc = nc
+
+dc = dc.where(dc['lat'] > -37, drop=True)
+dc = dc.where(dc['lat'] < 37, drop=True)
+dc = dc.where(dc['lon'] > -20, drop=True)
+dc = dc.where(dc['lon'] < 60, drop=True)
```

Comment:
Are these additive in some way? Just seems to overwrite the last line value with a new line value?